File: Obnovljivac.API/ClientApp/src/app/models/autogenerated/parameters.ts
import {ParameterInformation} from './parameterInformation';

// ----------------------------------------------
//  Interface and model
// ----------------------------------------------
export interface IParameters  {
    name: ParameterInformation;
}

export class Parameters  implements IParameters {
    public name: ParameterInformation;

    constructor(name: ParameterInformation = null ) {
        
        this.name = name;
    }
}

 

File: Obnovljivac.API/Models/NasaPower/Parameters.cs
namespace Obnovljivac.API.Models.NasaPower
{
    /// <summary>
    /// Model from 
    /// <see href="https://power.larc.nasa.gov/api/pages/?urls.primaryName=Daily">
    /// POWER Daily API
    /// </see>
    /// <br></br>
    /// The API allows daily data requests of POWER Analysis Ready Data (ARD).
    /// </summary>
    public class Parameters
    {
        public ParameterInformation Name { get; set; }
    }
}
